src/genius/event.components/life-event/life-event.component.html

```diff
@@ -1,26 +1,8 @@
-<ng-container [ngSwitch]="templateVersion">
-  <div *ngSwitchCase="lifeEventTemplateVersion.SHORTEST" class="life-event-shortest" (click)="returnLifeEvent(lifeEventTemplateAction.GET)">
+<div class="life-event-shortest-delete">
+  <div class="life-event-description" (click)="returnLifeEvent(lifeEventTemplateAction.GET)">
```

Comment:
use dedicated methods to do dedicated work: loadEvent for get, deleteEvent for delete

use everywhere same method is not optimization